core/baseline-core-v1.0-psd01.md

```diff
@@ -458,60 +462,89 @@ There MUST be a legally binding contract, however simple and temporary, before a
+
 #### **[CR1]<[D3]** 	
 There MUST be only one MSA between contract parties covering commercial transactions for a given set of products, services, or assets.
 
 *This requirement is intended to disambiguate which terms cover which part of a commercial relationship between parties.*
 
 *Specific Terms and Conditions (“Specific T&Cs”) defines the terms and conditions governing a specific product, service, or asset or set thereof offered and delivered by Provider(s) to Requester(s).*
 
+[[CR1]<[D3]](#cr1d3) Testability: Business logic can be expressed in software code, and software code can be tested, and since BPI logic can ensure only unique combinations of an MSA, unique identifier, and an agreement type exists between two or more counterparties, the requirement is testable.
+
 #### **[D4]**	
-Each specific product, service, or asset, or set thereof offered and delivered by Provider(s) to Requester(s) SHOULD have its Specific T&C document.
+Each specific product, service, asset, or set thereof offered and delivered by Provider(s) to Requester(s) SHOULD have its specific T&C document.
 
 *This would allow the fine-graining and consistent application of commercial State-Object-specific business rules and data.*
+
+[[D4]](#d4) Testability: Since a T&C document is part of a contract and a contract can be represented in a testable manner per the testability statement of [[R3]](#r3), requirement [[D4]](#d4) is testable.
+
 ### 2.4.2 Commercial Documents 
 
 Commercial documents, a category of commercial State Objects, refer to the state of a specific product/service/asset or set thereof, which may or may not be modified from an original offering to meet the Requester requirements and includes operational and commercial details. A commercial document is an abstract construct representing mutual commitments based on a legally binding contract.
 
 #### **[R4]**	
 A commercial State Object to be transacted on MUST be based on a specific commercial document.
 
+[[R4]](#r4) Testability: Given is an [Example](#functional-terms-implemented-as-zero-knowledge-circuit) of a specific commercial document necessary for a Commercial State Object to be transacted.
+
 #### **[R5]**  
 A commercial document MUST be derived from a legally binding contract.
 
+[[R5]](#r5) Testability: Given is an [Example](#commerical-agreement-as-verifiable-credential) of a legally binding contract. (legally binding contract can be expressed as given in example)
+
 #### **[R6]**	
 A commercial document MUST be represented as an electronic record on a BPI between the counterparties.
 
+[[R6]](#r6) Testability: Given is an [Example](#commerical-agreement-as-verifiable-credential) of a legally binding contract.
```

Comment:
```suggestion
[[R6]](#r6) Testability: Since the testability statement in [[R5]](#r5) represents an electronic version of a commercial document and is testable as stated, this requirement is testable as well.
```